ciao/arch/architecture.md

```diff
@@ -0,0 +1,422 @@
+---
+title: The Architecture of the Cloud Integrated Advanced Orchestrator (CIAO)
+permalink: architecture.html
+keywords: CIAO, architecture
+---
+
+## A whirlwind tour of the Cloud Integrated Advanced Orchestrator
+
+The Cloud Integrated Advanced Orchestrator (CIAO) is designed to be a simple
+IAAS solution for creating private clouds within an enterprise.  A CIAO
+cluster is composed from a number of different computing devices referred to as
+nodes.  It can be used to deploy units of compute, known as instances, across its
+nodes.  Once deployed, instances can be monitored and managed.  Two
+types of instances are supported; VM based instances, in which all computation
+is performed in a VM, and container instances, in which docker containers
```

Comment:
I'd spell out what is meant by VM here..it's hardware virtualisation vs a language VM

Reply:
I'm going to use hypervisor here, which should clarify things.